### ISSUE TITLE
A socket connected, but the session could not be loaded to pass to configured handler

### ISSUE THREAD

**Issue by Vadorequest** [balderdashy/sails#3000]
I get this weird message when I lift my app.

```js
warn: A socket connected, but the session could not be loaded to pass to configured handler: `sails.config.sockets.onConnect()`.  Will run handler with a fake, empty session.  Details:
 Error: Session could not be loaded
    at _createError (c:\wamp\www\Ayolan\Translation\node_modules\sails\lib\hooks\session\index.js:271:21)
    at Immediate._onImmediate (c:\wamp\www\Ayolan\Translation\node_modules\sails\lib\hooks\session\index.js:274:13)
    at processImmediate [as _immediateCallback] (timers.js:358:17) { [Error: Session could not be loaded] code: 'E_SESSION' }
```

It happens twice and then disappear, I think it may be because I restart the app while I have pages opened and connected by socket, which tries to reconnect ASAP but fails somehow when the server isn't ready.

I couldn't reproduce the issue if I lift while I don't have any opened pages, so I guess that's the reason.

I didn't have this issue using sails 0.10.

**Comment by Vadorequest** [comment]
By the way, the link for the documentation is dead.

```
  |>    Now connected to Sails.
\___/   For help, see: http://bit.ly/1DmTvgK
        (using browser SDK @v0.11.0)
```

It leads to http://sailsjs.org/#!/documentation/reference/BrowserSDK/BrowserSDK.html which... doesn't seem to exist.

**Comment by sailsbot** [comment]
Thanks for posting, @Vadorequest.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
